apps/site/assets/ts/schedule/schedule-loader.tsx

```diff
@@ -8,15 +8,23 @@ import { SchedulePageData } from "./components/__schedule";
 import { MapData, StaticMapData } from "../leaflet/components/__mapdata";
 import ScheduleFinder from "./components/ScheduleFinder";
 
-const renderMap = (): void => {
+const renderMap = ({
+  route_patterns: routePatternsByDirection,
+  direction_id: directionId
+}: SchedulePageData): void => {
+  const routePatterns = routePatternsByDirection[directionId];
+  const shapeIds = routePatterns.map(routePattern => routePattern.shape_id);
   const mapDataEl = document.getElementById("js-map-data");
   if (!mapDataEl) return;
```

Comment:
So, I'm not sure of the details of calling .render() on line 61 of MapTest.tsx, but this line appears to return from the function immediately if the map data element is not found, while that test only checks for thrown exceptions. Is there another test covering that?

Reply:
Yes, that should be covered by `ScheduleDirectionTest.tsx`, lines 281 and on. The map data element should be missing when we're using a static map image rather than the dynamic map we're used to, which IIRC we do only for ferry.

Reply:
Ok, thanks